src/utils/formatter.ts

```diff
@@ -76,3 +76,31 @@ export const formatOrgLogs = (allTasks: any[]) => {
 
   return obj
 }
+
+export const formatTxtLogs = (allTasks: string[]) => {
+  let obj: Record<string, formattedTaskType[]> = {}
+  let currentHeading: string = ''
+
+  allTasks.forEach((task: string) => {
+    const trimmedTask = task.trim()
+
+    // check if line starts with `#` followed by date
+    if (trimmedTask[0] === '#' && TXT_TIME_STAMP.test(task)) {
```

Comment:
```suggestion
    if (trimmedTask.startsWith('#') && TXT_TIME_STAMP.test(task)) {
```

Reply:
however it's an optional to change to this because we don't need any other complex checks